lib/App/cpanminus/script.pm

```diff
@@ -91,6 +91,7 @@ sub new {
         installed_dists => 0,
         install_types => ['requires'],
         with_develop => 0,
+        with_configure => undef,
```

Comment:
I think this is a bit complicated, and I would add `--with-configure` to enable configure when reading from cpanfile, `--without-configure` to disable that, which is default. And do not change the behavior for META files.